src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -398,7 +398,9 @@ def open_page_in_browser(url):
 
     if is_wsl():   # windows 10 linux subsystem
         try:
-            return subprocess.call(['cmd.exe', '/c', "start {}".format(url.replace('&', '^&'))])
+            # https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_powershell_exe
+            # Ampersand (&) should be quoted
+            return subprocess.call(['powershell.exe', '-Command', 'Start-Process "{}"'.format(url)])
```

Comment:
Without double quote, `powershell.exe` gives error:

```sh
# WSL
$ powershell.exe -Command 'Start-Process https://github.com/?a=1&b=2'
At line:1 char:38
+ Start-Process https://github.com/?a=1&b=2
+                                      ~
The ampersand (&) character is not allowed. The & operator is reserved for future use; wrap an ampersand in double
quotation marks ("&") to pass it as part of a string.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : AmpersandNotAllowed
```

This works:
```sh
$ powershell.exe -Command 'Start-Process "https://github.com/?a=1&b=2"'
```

Reply:
Could you please help share the reason why we change `cmd` to `powershell` ? Is there any problem with `cmd` ?

Reply:
Out of curiosity,  is it a powershell executable inside WSL?